src/django/api/serializers/v1/moderation_event_update_serializer.py

```diff
@@ -78,12 +84,58 @@ def __validate_status(self, value):
             })
         return value
 
+    def validate(self, data):
+        status = data.get('status')
+        if status == ModerationEvent.Status.REJECTED:
```

Comment:
Can we change this to be a reverse if statement something like:

```python
if  data.get('status') != ModerationEvent.Status.REJECTED:
  return data
```

Reply:
Done.